client/components/navbar.jsx

```diff
@@ -32,17 +32,21 @@ export default class Navbar extends React.Component {
 
     if (!user) return <Redirect to="sign-in" />;
 
+    if (user.profilePhotoUrl === null) {
+      user.profilePhotoUrl = '/placeholder-profile-image.jpeg';
+    }
```

Comment:
We generally don't modify state or props like this, you should declare a separate variable.